x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -223,6 +223,11 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables the new data ingestion hub
    */
   dataIngestionHubEnabled: false,
+
+  /**
+   * Enables the new Entity Store engine routes
+   */
+  entityStoreEngineRoutesEnabled: false,
```

Comment:
QQ: Are we going to use the same FF for the UI? 🤔

Reply:
I think so? Is there a reason why we'd have 2 separate flags?

Reply:
I'll rename it!